content/views.py

```diff
@@ -1,13 +1,12 @@
 from django.shortcuts import render
-from .models import OurStory, OurTeam, EduProgram, EthicalPost, VolunteerPeru, VolunteerOpenPosition, VolunteerAbout, CustomPage
+from .models import OurStory, OurTeam, EduProgram, EthicalPost, VolunteerPeru, VolunteerOpenPosition, VolunteerAbout, CustomPage,Home
 from django.utils import timezone
 
 
 def our_story(request):
     ourstory = OurStory.objects.all().order_by('order')
     return render(request, 'content/our_story.html', {'ourstory': ourstory})
 
-
```

Comment:
I agree that these are unnecessary but its good cleanup. The original PR that had those extra lines should have not added those